### ISSUE TITLE
I am not able to get the info for the bowen ferry

### ISSUE THREAD

**Issue by enejb** [samuel-pratt/ferrytimes-api#1]
Are you able to get the info for the return trip as well? 

The URL that I am trying to request is 

https://ferrytimes.ca/api/horseshoe-bay/snug-cove-bowen-island which seems to return a 500
and 
https://www.api.ferrytimes.ca/api/horseshoe-bay/snug-cove-bowen-island/ also return a 500 error.

Not sure if the urls should be something different.

**Comment by samuel-pratt** [comment]
This was an error I made in the readme, I've updated the Bowen Island key to be "snug-cove-(bowen-is.)". I'm hoping to update the api to use easier to write urls in the future but for now it is easiest to list it how BC Ferries does on their site.

As for the return trip, BC Ferries does not list capacities or current updates for departures from Bowen Island so I didn't think to include it. I'll update the api soon to include departures from Bowen Island.
